### ISSUE TITLE
model isnt't updated upon ESC in IE

### ISSUE THREAD

**Issue by kamilkp** [angular/angular.js#7297]
This happens in Internet Explorer 11. Hitting ESC in an `input` element clears it but the model isn't updated.

Fiddle: http://jsfiddle.net/e8TDg/
type something in the `input` and then hit ESC button on keyboard. The `input` clears, but the model value remains.

**Comment by Splaktar** [comment]
I'm not sure if this was fixed by Microsoft as they are no longer updating IE11. They also shut down Microsoft Connect, so the bug link above no longer works.

It also looks like JSFiddle no longer supports IE11 (it's just a bouncing cloud forever).

But I created a [CodePen](https://s.codepen.io/Splaktar/debug/EeNVNj) for this and testing in IE11 seems to demonstrate that the model is now updated when the `ESCAPE` key clears the input. Verified this with AngularJS 1.7.2.

CC @gkalpak
